CodeSniffer/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -33,6 +33,34 @@ class Squiz_Sniffs_Commenting_FunctionCommentSniff extends PEAR_Sniffs_Commentin
 {
 
     /**
+     * The number of spaces after the type of a parameter.
+     *
+     * @var int
+     */
+    public $spacingAfterParamType = 1;
+
+    /**
+     * The number of spaces after the name of a parameter.
+     *
+     * @var int
+     */
+    public $spacingAfterParamName = 1;
+
+    /**
+     * Check that param comments start with a capital letter.
+     *
+     * @var boolean
+     */
+    public $paramCommentCapitalStart = true;
+
+    /**
+     * Check that param comments end with the full stop.
+     *
+     * @var boolean
+     */
+    public $paramCommentFullStop = true;
```

Comment:
AFAIK a full stop is not a mandatory requirement, but I think it is useful to make this setting customizable in either way, as there are many projects who do not use them (e.g. Symfony, CodeIgniter, etc.)